tests/unittests/utils.py

```diff
@@ -0,0 +1,49 @@
+import os
+import tempfile
+import yaml
+import sqlite3
+import pytest
+from qe_metrics.libs.database import Database
+
+TEMP_DB_FILE = "/tmp/qe_metrics_test.sqlite"
+
+
+def temp_db_creds():
+    creds = {
+        "database": {
+            "local": True,
+            "local_filepath": TEMP_DB_FILE,
+        }
+    }
+
+    with tempfile.NamedTemporaryFile("w", delete=False, suffix=".yaml") as temp_creds:
```

Comment:
```suggestion
    with tempfile.NamedTemporaryFile("w", suffix=".yaml") as temp_creds:
```
use yield so the file will not be deleted until teardown